flexible-classroom/get-started/enable-flexible-classroom.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: "Enable Flexible Classroom"
+sidebar_position:  2
+type: docs
+description: >
+   Enable the Flexible Classroom service in Agora Console.
+---
+
+import Prerequites from '@docs/shared/common/prerequities.mdx';
+
+export const toc = [{}];
+
+This page introduces how to enable the Flexible Classroom service in Agora Console.
+
+In order to follow this procedure you must have:
```

Comment:
Suggestion: For lines 15 to 17, are not related to this page. Please remove it.
Reason: Wrong description.

Reply:
:-D, nice spot. Removed